### ISSUE TITLE
[FEATURE] Add constructor with value to Reference classes

### ISSUE THREAD

**Issue by KurtGokhan** [AdamRamberg/unity-atoms#112]
**Is your feature request related to a problem? Please describe.**

It is common sense to write default values to your fields in MonoBehaviors. Normally you would do it like:

```
public float WalkSpeed = 15;
```

So we should be able to do this with Atoms:
```
public FloatReference WalkSpeed = new FloatReference(15);
```

**Describe the solution you'd like**

The default and generated Reference classes should have a constructor that accepts a constant value.

**Describe alternatives you've considered**

Adding these lines manually to the source file in the package, or generate new classes and add it manually. 

```
        public FloatReference() { }
        public FloatReference(float value) : base(value) { }
```

**Additional context**
It seems easy to do. But if someone uses Reflection to access constructor, there may be errors as there will be 2 constructors after implementing this.

**Comment by AdamRamberg** [comment]
Thanks for reporting this issue! Sounds like a great feature. Will add that to the 3.0 milestone.

**Comment by AdamRamberg** [comment]
Implemented in #113
